packages/hooks/src/index.ts

```diff
@@ -1,4 +1,3 @@
-// Export all hooks
 export * from "./useAssert";
```

Comment:
Stop this from being made on every branch. This could have been a cleanup PR but it is literally 1 line